src/main.rs

```diff
@@ -14,12 +14,36 @@ fn main() -> Result<(), Box<dyn Error>> {
         process::exit(0);
     }
 
+    let is_fix = args.is_present("fix");
+
     let warnings = dotenv_linter::run(args, &current_dir)?;
-    if warnings.is_empty() {
-        process::exit(0);
+
+    if is_fix {
+        if warnings.iter().any(|w| w.is_fixed) {
+            println!("Fixed warnings:");
+            warnings
+                .iter()
+                .filter(|w| w.is_fixed)
+                .for_each(|w| println!("{}", w));
+        }
+
+        if warnings.iter().any(|w| !w.is_fixed) {
+            println!("\nUnfixed warnings:");
+            warnings
+                .iter()
+                .filter(|w| !w.is_fixed)
+                .for_each(|w| println!("{}", w));
+        } else {
+            process::exit(0);
+        }
+    } else {
+        if warnings.is_empty() {
```

Comment:
I think it might be before the parent condition block, if warnings are empty - that's always ok.

Reply:
"No warnings" is the "default" (likely) flow, so I made this change.